src/scraper/scraper.cpp

```diff
@@ -708,7 +708,7 @@ void Scraper(bool bSingleShot)
     {
         pathDataDir = GetDataDir();
         // This is necessary to maintain compatibility with Windows.
-        pathDataDir.imbue(std::locale(std::locale(), new std::codecvt_utf8_utf16<wchar_t>()));
+        // pathDataDir.imbue(std::locale(std::locale(), new std::codecvt<wchar_t, char, std::mbstate_t>));
```

Comment:
Consider removing these and their comments if we don't need them anymore.

Reply:
Yeah I forgot! :)

Reply:
Done.